src/Components/v2/CarouselV2.tsx

```diff
@@ -0,0 +1,265 @@
+
+  let flickity = null
+
+  const checkLastItemVisible = () => {
+    if (!flickity.slides.length) {
+      return
+    }
+    const isLastItemVisible = flickity.selectedElements.includes(
+      flickity.getLastCell().element
+    )
+    setLastItemVisible(isLastItemVisible)
+  }
+
+  if (isMounted) {
+    if (flicktyRef.current !== null) {
+      flickity = flicktyRef.current.flkty
+    }
+
+    flickity.on("select", index => {
+      setSlideIndex(index)
+      checkLastItemVisible()
+    })
+  }
+
+  const settings = {
+    draggable: false,
+    freeScroll: false,
+    wrapAround: false,
+    cellAlign: "left",
+    pageDots: false,
+    prevNextButtons: false,
+    groupCells: true,
+    contain: true,
+    ...props.settings,
+  }
+
+  const LeftArrow = () => {
+    const getFlickity = flickity
+    return (
+      <ArrowButton
+        onClick={() => {
+          getFlickity && getFlickity.previous ? getFlickity.previous() : null // check existence for tests
+        }}
+      >
+        <ChevronIcon direction="left" fill="black100" width={30} height={30} />
+      </ArrowButton>
+    )
+  }
+
+  const RightArrow = () => {
+    const getFlickity = flickity
+    return (
+      <ArrowButton
+        onClick={() => {
+          getFlickity && getFlickity.next ? getFlickity.next() : null // check existence for tests
+        }}
+      >
+        <ChevronIcon direction="right" fill="black100" width={30} height={30} />
+      </ArrowButton>
+    )
+  }
+  return (
+    <Flex
+      flexDirection="row"
+      position="relative"
+      justifyContent="space-around"
+      alignItems="center"
+      height={props.height}
+    >
+      {(currentSlideIndex !== 0 || settings.wrapAround === true) && (
+        <ArrowWrapper left={-38}>
+          {props.renderLeftArrow ? (
+            props.renderLeftArrow({
+              Arrow: LeftArrow,
+              getFlickity: flickity,
+              ...props,
+            })
+          ) : (
+            <LeftArrow />
+          )}
+        </ArrowWrapper>
+      )}
+
+      <CarouselContainer height={props.height}>
+        {Flickity && (
+          <Flickity options={settings} ref={flicktyRef}>
+            {props.data.map((slide, index) => {
+              return <Box key={index}>{props.render(slide)}</Box>
+            })}
+          </Flickity>
+        )}
+      </CarouselContainer>
+
+      {(!lastItemVisible || settings.wrapAround === true) && (
+        <ArrowWrapper right={-38}>
+          {props.renderRightArrow ? (
+            props.renderRightArrow({
+              Arrow: RightArrow,
+              getFlickity: flickity,
+              ...props,
+            })
+          ) : (
+            <RightArrow />
+          )}
+        </ArrowWrapper>
+      )}
+    </Flex>
+  )
+}
+
+export const SmallCarousel = (props: Props) => {
+  const settings = {
+    draggable: true,
+    freeScroll: false,
+    wrapAround: false,
+    cellAlign: "left",
+    pageDots: false,
+    prevNextButtons: false,
```

Comment:
These settings are all specific to Flickity, yes? (Wondering if any are left over from slick)

Reply:
All Flickity settings.